### DISCUSSION TITLE
Option to choose a new upload folder while keeping the existing folder attached

### DISCUSSION THREAD

**Discussion by giripriyadarshan** [immich-app/immich#6883] [General]
This will enable horizontal scaling of media storage as we can choose a new folder on a new drive to upload when the existing drive runs out of memory...... External folders option doesnt allow syncing, so for media sync this feature will be useful

**Comment by bo0tzz** [comment]
As mentioned in #6882, spreading storage across disks is not Immich's concern. You should use an appropriate technology (RAID, ZFS, etc) to combine multiple drives instead.
